command_line/symmetry.py

```diff
@@ -224,6 +225,39 @@ def eliminate_sys_absent(experiments, reflections):
     return reflections
 
 
+def get_subset_for_symmetry(params, experiments, reflection_tables):
```

Comment:
Given this function only uses the `exclude_images` attribute of `params`, why not just directly pass in `exclude_images` as a paramer instead of `params`?